README.md

```diff
@@ -1 +1,75 @@
-# awesome-book-es6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# AWESOME BOOKS WITH ES6
+
+## Description
+
+This is an awesome application that allows users to save/add list of books (title, author)
+
+
+## Build With
+
+- Editor : Visual studio code
+
+- Languages
+
+  - [ ] HTML
+  - [ ] JavaScript
+
+- Terminologies
+  - [ ] Github
+  - [ ] Linters
+  - [ ] WebHint
+  - [ ] Stylelint
+  - [ ] ESlint
+
+<!-- ## Live Demo 
+
+[Live Demo Link](https://htetaungkyaw71.github.io/awesome-books/) -->
+
+<!-- ## Link to Presentation on this Project
+
+[See Presentation]() -->
+
+
+## Getting Started
+
+step 1:
+Clone this repo locally using git in the command line and the following command
+
+```
+git clone git@github.com:Htetaungkyaw71/awesome-book-es6.git
+```
+
+Alternatively, you can just download the complete zip file and extract the folder in your directory
+
+Step 2:
+Copy All Project files and run using VSCODE. HTML can be viewed in browser.
+
+
+
+## Author 1
+
+👤 **Htetaungkyaw**
+
+- GitHub: [@githubhandle](https://github.com/Htetaungkyaw71)
+- Twitter: [@twitterhandle](https://twitter.com/htetaun91907337)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/htet-aung-kyaw-9a77271a7/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Htetaungkyaw71/awesome-book-es6/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- ### Original design idea by **Cindy Shin in Behance**, in association with the Creative Commons [license](./CC.md) of the design.
+
+## 📝 License
+
+This project is [MIT](..Licence) licensed.
```

Comment:
- [OPTIONAL] The MIT link is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯